cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -152,195 +153,115 @@ defaultConwayGenesis =
 
 -- | Configuration value that allows you to hardfork to any Cardano era
 -- at epoch 0.
-defaultYamlHardforkViaConfig :: ShelleyBasedEra era -> KeyMapAeson.KeyMap Aeson.Value
+defaultYamlHardforkViaConfig :: ShelleyBasedEra era -> Aeson.KeyMap Aeson.Value
 defaultYamlHardforkViaConfig sbe =
-  mconcat $ concat
-    [ defaultYamlConfig
-    , tracers
-    , protocolVersions sbe
-    , hardforkViaConfig sbe
-    ]
-
+  defaultYamlConfig
+    <> tracers
+    <> protocolVersions sbe
+    <> hardforkViaConfig sbe
  where
   -- The protocol version number gets used by block producing nodes as part
   -- of the system for agreeing on and synchronising protocol updates.
   -- NB: We follow the mainnet protocol versions and assume the latest
   -- protocol version for a given era that has had an intraera hardfork.
-  protocolVersions :: ShelleyBasedEra era -> [KeyMapAeson.KeyMap Aeson.Value]
+  protocolVersions :: ShelleyBasedEra era -> Aeson.KeyMap Aeson.Value
   protocolVersions sbe' =
-    case sbe' of
-      ShelleyBasedEraShelley ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 2)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
-      ShelleyBasedEraAllegra ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 3)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
-      ShelleyBasedEraMary ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 4)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
-      ShelleyBasedEraAlonzo ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 5)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
-      ShelleyBasedEraBabbage ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 8)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
-      ShelleyBasedEraConway ->
-        map (uncurry KeyMapAeson.singleton)
-          [ ("LastKnownBlockVersion-Major", Aeson.Number 9)
-          , ("LastKnownBlockVersion-Minor", Aeson.Number 0)
-          , ("LastKnownBlockVersion-Alt", Aeson.Number 0)
-          ]
+    Aeson.fromList
+      [case sbe' of
```

Comment:
I'm not so sure which one is better here. I don't remember Minor and Alt values being used here though.

Reply:
> I don't remember Minor and Alt values being used here though

Yeah that's why I shared their definitions here. There was no tuning of those values in any era.